radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/SimulationTest.java

```diff
@@ -479,112 +509,97 @@ public void configure() {
 			modules.add(new MockedSystemModule());
```

Comment:
This method is really problematic.  Even though it might have been made shorter with this PR, it's still 136 lines long and needs some attention.

One of the easy things that could be done here would be to separate the module data from the selection logic.